docs/knobs.md

```diff
@@ -0,0 +1,100 @@
+# Percona knobs variables
+
+Percona Server for MongoDB makes several additional configuration values
+available on the command line as well as in configuration files.
+
+## cursorTimeoutMillis
+
+Command line:
+
+```
+--cursorTimeoutMS {integer}
+```
+
+Configuration File:
+
+```
+Percona:
+  cursorTimeoutMillis: {integer}
+```
+
+Default value: 600000 (Ten Minutes)
+
+Sets the duration of time after which idle query cursors will be removed from
+memory.
+
+## failIndexKeyTooLong
+
+Command Line:
+```
+--failIndexKeyTooLong {true|false}
+```
+
+Configuration File:
+```
+Percona:
+  ignoreLongIndexError: {true|false}
+```
+
+Default value: true 
```

Comment:
I think the default for `ignoreLongIndexError` is `false` while the default for command line option is `true`. This corresponds to their names: one is `ignore` and other is `fail`.